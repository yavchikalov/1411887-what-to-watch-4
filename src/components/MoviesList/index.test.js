import React from 'react';
import {MoviesList} from './';
import renderer from 'react-test-renderer';

const movies = [
  {
    name: `Fantastic Beasts`,
    cover: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    trailer: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  },
  {
    name: `Bohemian Rhapsody`,
    cover: `img/bohemian-rhapsody.jpg`,
    trailer: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  },
  {
    name: `Macbeth`,
    cover: `img/macbeth.jpg`,
    trailer: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  }
];

describe(`MoviesList`, () => {
  it(`snapshot MoviesList`, () => {
    const tree = renderer.create(
        <MoviesList
          movies={movies}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
