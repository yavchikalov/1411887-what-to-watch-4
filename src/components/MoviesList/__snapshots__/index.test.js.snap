// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesList snapshot MoviesList 1`] = `
Array [
  <article
    className="small-movie-card catalog__movies-card"
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        onMouseOut={[Function]}
        onMouseOver={[Function]}
        poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
        src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Fantastic Beasts
      </a>
    </h3>
  </article>,
  <article
    className="small-movie-card catalog__movies-card"
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        onMouseOut={[Function]}
        onMouseOver={[Function]}
        poster="img/bohemian-rhapsody.jpg"
        src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Bohemian Rhapsody
      </a>
    </h3>
  </article>,
  <article
    className="small-movie-card catalog__movies-card"
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        onMouseOut={[Function]}
        onMouseOver={[Function]}
        poster="img/macbeth.jpg"
        src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Macbeth
      </a>
    </h3>
  </article>,
]
`;
